ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/picklist/activity-picklist-remote-auto-complete-options.component.ts

```diff
@@ -114,4 +118,13 @@ export class ActivityPicklistRemoteAutoCompleteOptionsComponent
         this.block.answer = value ? [{ stableId: value.stableId, detail }] : [];
         this.valueChanged.emit([...this.block.answer]);
     }
+
+    getInitialAnswerOptionLabel(): string {
+        if (
+            this.block.selectMode === 'SINGLE' &&
+            this.block.picklistOptions.length
+        ) {
+            return this.block.picklistOptions[0].optionLabel;
```

Comment:
Actually, it is a little more complicated than this.
Need to deal with case where the answer that was saved is just the text entered.
Remember the example:
What fruit do you like?
Apples
Oranges
Other ___________

You are not handling the case where somebody typed an option.
The text would come in that details property in the answer that we have been talking about.

From the very similar component, this is how she was initializing the answer:

https://github.com/broadinstitute/ddp-angular/blob/cfffe1b69f2f9f8c190978f3d9ba3673719d929c/ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/picklist/autocompleteActivityPicklistQuestion.component.ts#L138-L147

Reply:
I guess your code was not looking for situation where user types an option that does not exist.
Behavior we want is to include as stableid an option that has a property of allowDetails = true and include in the answer the property detail: THEVALUEENTERED.
Not seeing this at all in the code.
But I get the impression that our remote API does not account for it.
@ssettipalli Can you work with Jose to nail down these details?